redis/Dockerfile

```diff
@@ -1,12 +1,29 @@
-FROM clearlinux:latest
+FROM clearlinux:latest AS builder
+
+# Move to latest Clear Linux release to ensure
+# that the swupd command line arguments are
+# correct
+RUN swupd update --no-boot-update $swupd_args
+
+# Grab os-release info from the minimal base image so
+# that the new content matches the exact OS version
+COPY --from=clearlinux/os-core:latest /usr/lib/os-release /
```

Comment:
I think we could get into a weird position where the swupd os-install below fails if the clearlinux/os-core:latest is not yet updated when this unit is updated. It is likely safer to source the os-release from os-core first and run update with -V ${VERSION_ID}. I will say this problem is more theoretical for our Dockerfiles but if other folks reuse the pattern they could more readily run into trouble.

Reply:
So you mean something on the lines of...
COPY --from=clearlinux/os-core:latest /usr/lib/os-release /
RUN source /os-release && swupd update --no-boot-update $swupd_args

Reply:
Yes ```RUN source /os-release && swupd update --no-boot-update -V ${VERSION_ID} $swupd_args```